nautobot_circuit_maintenance/templates/nautobot_circuit_maintenance/circuitimpact.html

```diff
@@ -1,4 +1,4 @@
-{% extends 'base.html' %}
+{% extends 'generic/object_detail.html' %}
```

Comment:
`object_detail.html` was only added in Nautobot 1.2 so this change will need to be accompanied by an increase in the project's required Nautobot min_version.